packages/backend-modules/discussions/graphql/resolvers/DiscussionsStats/evolution.js

```diff
@@ -1,20 +1,10 @@
 const moment = require('moment')
 
-const { createCache } = require('../../../lib/stats/evolution')
-
 module.exports = async (_, args, context) => {
   // Fetch pre-populated data
-  const data = await createCache(context).get()
-
-  // In case pre-populated data is not available...
-  if (!data) {
-    throw new Error(
-      'Unable to retrieve pre-populated data for DiscussionsStats.evolution',
-    )
-  }
-
-  // Retrieve pre-populated data.
-  const { result = [], updatedAt = new Date() } = data
+  const data = await context.pgdb.query(
+    'select * from cockpit_discussions_evolution;',
+  )
```

Comment:
for the other data you added a check that throws an error if the data is not there, I'd do that here as well :)

Reply:
@lucianak I change the behaviour of the other resolvers to don't throw errors if there is no data.